etc/benchmarks/bson_versions.json

```diff
@@ -0,0 +1,10 @@
+{
+  "versions": [
+    "1.1.6",
+    "4.6",
+    "5.0",
+    "5.1",
+    "5.2",
+    "5.3"
+  ]
+}
```

Comment:
Note that the versions here are not special. These will likely be updated once the regression tests are added